### PR TITLE
Minor additions to the documentation about higher order output.

### PR DESCRIPTION
**Pull request by bangerth**
Part of #7021.

### CODE REVIEW

File: include/deal.II/numerics/data_out.h [hunk 1]
```diff
@@ -77,9 +77,10 @@ namespace internal
  * which are the objects that are later output by the functions of the base
  * classes. You can give a parameter to the function which determines how many
  * subdivisions in each coordinate direction are to be performed, i.e. of how
- * many subcells each patch shall consist. Default is one, but you may want to
- * choose a higher number for higher order elements, for example two for
- * quadratic elements, three for cubic elements three, and so on. The purpose
+ * many subcells each patch shall consist. The default is one, but you may want
+ * to choose a higher number for higher order elements, for example two for
+ * quadratic elements, three for cubic elements three, and so on. (See
```
Comment (kronbichler) [inline include/deal.II/numerics/data_out.h]:
```suggestion
 * quadratic elements, three for cubic elements, and so on. (See
```

**Comment by bangerth** [comment]
I fixed the (pre-existing) typo -- thanks for reading more carefully than I did.

**Comment by kronbichler** [comment]
@bangerth it seems we cannot get Travis back running by other means than rebasing - can you give it try?

**Comment by bangerth** [comment]
Rebased.

**Pull request merged**